solr/modules/jwt-auth/src/java/org/apache/solr/security/jwt/JWTAuthPlugin.java

```diff
@@ -847,9 +851,11 @@ protected String generateAuthDataHeader() {
     Map<String, Object> data = new HashMap<>();
     data.put(
         JWTIssuerConfig.PARAM_AUTHORIZATION_ENDPOINT, primaryIssuer.getAuthorizationEndpoint());
+    data.put(JWTIssuerConfig.PARAM_TOKEN_ENDPOINT, primaryIssuer.getTokenEndpoint());
```

Comment:
In the `X-Solr-AuthData` the keys should be "snake_case", so this should be `token_endpoint`, not `tokenEndpoint`. The frontend part will probably need to adapt to this as well.
```suggestion
    data.put("token_endpoint", primaryIssuer.getTokenEndpoint());
```